.github/workflows/assign_qa_to_issue.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Checks if health/global/design-system label is applied and then assigns the correct QA team member to that ticket
+#
+
+name: '[Utils] assign qa to issue'
+
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  run_on_labeled:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: if_health
+        if: github.event.label.name == 'Health'
+        uses: actions-ecosystem/action-add-assignees@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          assignees: "DJUltraTom"
```

Comment:
Assigning people to issues can also be done via the GH CLI with a one-liner

```yaml
gh issue edit ${{ github.event.issue.id }} --add-assignee "DJUltraTom"
```

That way we won't have to rely on a 3rd party action

Reply:
updated, good call